app/models/round.py

```diff
@@ -24,8 +43,16 @@ class Round:
     feedback_link: str
     project_name_field_id: str
     application_guidance: str
-    requires_feedback: bool = False
     mark_as_complete_enabled: bool = False
+    feedback_survey_config: FeedbackSurveyConfig = None
+
+    def __post_init__(self):
+        if isinstance(self.feedback_survey_config, dict):
+            self.feedback_survey_config = FeedbackSurveyConfig.from_dict(
+                self.feedback_survey_config
+            )
+        elif self.feedback_survey_config is None:
+            self.feedback_survey_config = FeedbackSurveyConfig()
```

Comment:
Do we need this? Couldn't we just have `= FeedbackSurveyConfig()` instead of `= None`?

Reply:
Made to use default values here, if it's None